addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -564,6 +567,11 @@ def _l10n_es_edi_call_web_service_sign(self, invoices, info_list):
                 results[inv] = {'success': True}
                 inv.message_post(body=_("We saw that this invoice was sent correctly before, but we did not treat "
                                         "the response.  Make sure it is not because of a wrong configuration."))
+
+            elif respl.CodigoErrorRegistro == 1117 and not self.env.get('error_1117'):
```

Comment:
should not it be self.env.context.get?  Are you sure the CodigoErrorRegistro is an int?  
```suggestion
            elif respl.CodigoErrorRegistro == 1117 and not self.env.context.get('error_1117'):
```
I still think you could try with another existing error to at least see that everything works fine.  @yosa-odoo

Reply:
If I go with another flow, the error message is an INT yes
![image](https://user-images.githubusercontent.com/94046269/197742354-eafd9873-b267-4410-889f-fd303db387c8.png)

Reply:
> I still think you could try with another existing error to at least see that everything works fine

Well, there will be no 1117 error in a Test Env, there will be duplicated error.